src/OpenID/Connect/Authentication.hs

```diff
@@ -183,3 +186,18 @@ data ClientAuthentication
 
   deriving stock (Generic, Eq, Show)
   deriving (ToJSON, FromJSON) via GenericJSON ClientAuthentication
+
+data TokenEndpointAuthMethod
+  = StandardAuthentication ClientAuthentication
+  | AuthenticationExtension Text
+
```

Comment:
Please remove this space so the formatting matches the reset of the code.  Eventually I'll come back and reformat all the source code with Ormolu but until then...